### PR TITLE
Add `tiup ctl pd` command in tiup page.

### PR DESCRIPTION
**Pull request by LittleFall**
### What is changed, added or deleted? (Required)

1. Swap lines to stress the `Note` part.
2. Use `tiup ctl pd` instead of `pd-ctl` in tiup page.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
@Little-Wallace If a PR applies to master and other release branches, the best practice is to create the PR based on the master branch.

**Review by JaySon-Huang — COMMENTED** [review]
LGTM with the changes.
And you should reset the target branch of this PR to merge it to master.

**Comment by TomShawn** [comment]
> LGTM with the changes.
> And you should reset the target branch of this PR to merge it to master.

We can use the `needs-cherry-pick-master` label to cherry-pick to master, but this is not the best practice.

File: scale-tidb-using-tiup.md [hunk 1]
```diff
@@ -126,14 +126,14 @@ tiup cluster display <cluster-name>
 
 ## 扩容 TiFlash 节点
 
+如果要添加一个 TiFlash 节点，IP 地址为 10.0.1.4，可以按照如下步骤进行操作。
```
Comment (TomShawn) [inline scale-tidb-using-tiup.md:129]:
```suggestion
如果要添加一个 TiFlash 节点，其 IP 地址为 `10.0.1.4`，可以按照如下步骤进行操作。
```

File: scale-tidb-using-tiup.md [hunk 2]
```diff
@@ -126,14 +126,14 @@ tiup cluster display <cluster-name>
...
+
 > **注意：**
 >
 > 在原有 TiDB 集群上新增 TiFlash 组件需要注意：
 >
 > 1. 首先确认当前 TiDB 的版本支持 TiFlash，否则需要先升级 TiDB 集群至 4.0 rc 以上版本。
```
Comment (TomShawn) [inline scale-tidb-using-tiup.md:135]:
```suggestion
> 1. 首先确认当前 TiDB 的版本支持 TiFlash，否则需要先升级 TiDB 集群至 4.0 RC 以上版本。
```

File: scale-tidb-using-tiup.md [hunk 3]
```diff
@@ -126,14 +126,14 @@ tiup cluster display <cluster-name>
...
-> 2. 下载 [pd-ctl](https://download.pingcap.org/tidb-v4.0.0-rc.2-linux-amd64.tar.gz)，执行 `config set enable-placement-rules true` 命令，以开启 PD 的 Placement Rules 功能。
-
-如果要添加一个 TiFlash 节点，IP 地址为 10.0.1.4，可以按照如下步骤进行操作。
+> 2. 执行 `tiup ctl pd -u <pd-host>:<pd-port> config set enable-placement-rules true`，以开启 PD 的 Placement Rules 功能。或通过 [pd-ctl](https://download.pingcap.org/tidb-v4.0.0-rc.2-linux-amd64.tar.gz) 执行对应命令。
```
Comment (TomShawn) [inline scale-tidb-using-tiup.md:136]:
```suggestion
> 2. 执行 `tiup ctl pd -u <pd-host>:<pd-port> config set enable-placement-rules true` 命令，以开启 PD 的 Placement Rules 功能。或通过 pd-ctl（[下载地址](https://download.pingcap.org/tidb-v4.0.0-rc.2-linux-amd64.tar.gz)）执行对应的命令。
```

File: scale-tidb-using-tiup.md [hunk 4]
```diff
@@ -126,14 +126,14 @@ tiup cluster display <cluster-name>
...
 > **注意：**
 >
 > 在原有 TiDB 集群上新增 TiFlash 组件需要注意：
 >
-> 1. 首先确认当前 TiDB 的版本支持 TiFlash，否则需要先升级 TiDB 集群至 4.0 rc 以上版本。
-> 2. 下载 [pd-ctl](https://download.pingcap.org/tidb-v4.0.0-rc.2-linux-amd64.tar.gz)，执行 `config set enable-placement-rules true` 命令，以开启 PD 的 Placement Rules 功能。
-
-如果要添加一个 TiFlash 节点，IP 地址为 10.0.1.4，可以按照如下步骤进行操作。
+> 1. 首先确认当前 TiDB 的版本支持 TiFlash，否则需要先升级 TiDB 集群至 4.0 RC 以上版本。
+> 2. 执行 `tiup ctl pd -u <pd-host>:<pd-port> config set enable-placement-rules true` 命令，以开启 PD 的 Placement Rules 功能。或通过 pd-ctl（[下载地址](https://download.pingcap.org/tidb-v4.0.0-rc.2-linux-amd64.tar.gz)）执行对应的命令。
```
Comment (TomShawn) [inline scale-tidb-using-tiup.md:136]:
```suggestion
> 2. 执行 `tiup ctl pd -u <pd-host>:<pd-port> config set enable-placement-rules true` 命令，以开启 PD 的 Placement Rules 功能。或通过 [pd-ctl](/pd-control.md) 执行对应的命令。
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**